src/components/header/Header.js

```diff
@@ -0,0 +1,17 @@
+import HtmlElement from '../htmlElement/HtmlElement';
+import './styles.css';
+
+export class Header extends HtmlElement {
+  constructor(label = '', classList = [], htmlTag = null) {
```

Comment:
Maybe `label` could also be an optional parameter of the constructor of `HtmlElement`? You could then:
```
export class HtmlElement {
  constructor(classList = [], htmlTag = null, label='') {
    this.classList = classList;
    this.htmlTag = htmlTag;
    this.label = label
  }

  render() {
    const htmlElement = document.createElement(this.htmlTag);

    if (Array.isArray(this.classList)) {
      this.classList.forEach((element) => {
        htmlElement.classList.add(element);
      });
    }

    if (label) htmlElement.innerText = this.label

    return htmlElement;
  }
}
```